test/k8sT/Policies.go

```diff
@@ -801,7 +808,7 @@ var _ = Describe("K8sPolicyTest", func() {
 				appPods         map[string]string
 				app1PodIP       string
 				bindManifest    string
-				worldTarget     = "http://world1.cilium.test"
+				worldTarget     = "http://fqdn-httpd-svc.default.svc.cluster.local"
```

Comment:
Shouldn't this be `"http://fqdn-httpd-svc."+DefaultNamespace+".svc.cluster.local"`

Reply:
That hadn't even occurred to me but yes, 200%